content/events/2024-01-10-mastering-bitcoin-whitepaper.md

```diff
@@ -0,0 +1,58 @@
+
+Discover, discuss, and delve into the world of Bitcoin with us. Your journey towards mastering Bitcoin starts here!
+
+## How It Works
+
+On your own time, read this week's chapter. Show up at this event (in-person or remotely) to discuss the chapter with other participants. Each session will follow this format:
+
+1. Introductions: participants are invited to introduce themselves (we will skip this if there is nobody new in the room).
+2. Queries: then, we will go around the room to quickly discern what questions people had, what people were confused about, etc. and take note of these on the whiteboard so we can discuss further.
+3. Discussion: next, we will try and collectively answer the study questions together, as well as address any issues people defined in the prior step.
+4. Chill: afterwards, those joining in-person typically go to a restaurant or bar together to socialize.
+
+## This Session
+
+In order to give people time to obtain a copy of the book, we are going to first start with the <a href="https://spiral.xyz/bitcoin.pdf" target="_blank">Bitcoin Whitepaper</a>, which can be found as an appendix in the book, but also freely available on the internet. <a href="https://spiral.xyz/bitcoin.pdf" target="_blank">Read it </a>here.
+
+### Study Questions
+
+These questions are provided to help guide your thinking as read the material. We'll try to answer these together when we meet to discuss the chapter.
+
+1. What problem does Bitcoin aim to solve in the context of electronic cash systems, and how does it propose to solve it?
+2. Explain the concept of a peer-to-peer network in the context of Bitcoin and its significance in ensuring the security and functionality of the system.
+3. Discuss the role of digital signatures in Bitcoin transactions. How do they contribute to the integrity and security of the system?
+4. What is double-spending in the context of digital currencies, and how does Bitcoin address this issue?
+5. Describe the proof-of-work system used in Bitcoin. How does it contribute to the security and trust of the blockchain?
+6. How does Bitcoin ensure the validity of transactions and prevent fraudulent activities without relying on a central authority?
+7. Discuss the incentives for participants in the Bitcoin network. How do these incentives ensure the longevity and stability of the system?
+8. Explore the concept of privacy in Bitcoin transactions. How does the system maintain a balance between transparency and user privacy?
+9. Reflect on the potential implications of Bitcoin on the future of financial transactions and digital currencies.
+
+---
+
+## Frequently Asked Questions
+
+### Do I have to buy the book?
+
```

Comment:
dang... the github isn't updated? that blows

Reply:
Yup, but it will be one day